index.js

```diff
@@ -252,6 +259,32 @@ function setCookie (res, name, val, options) {
   res.setHeader('set-cookie', header)
 }
 
+/**
+ * Get secrect from set-cookie header in HTTP response.
+ *
+ * @param {OutgoingMessage} res
+ * @param {Object} [cookie]
+ * @api private
+ */
+
+function getSecertFromSetCookieHeader (res, cookie) {
+  let setCookieHeaders = res.getHeader("set-cookie")
+  if (setCookieHeaders) {
+    for (let setCookieHeader of setCookieHeaders) {
+      setCookieHeader = Cookie.parse(setCookieHeader)
+      if (cookie.Path !== setCookieHeader.path) {
+        continue;
+      }
+      if (!cookie.Domain) {
+        return setCookieHeader[cookie.key]
+      }
+      if (cookie.Domain && cookie.Domain === setCookieHeader.domain) {
```

Comment:
The `domain` attribute name is not case-sensitive, so it should not be treating it as such. In addition, the domain matching here is not an exact match; you can find the domain matching algorithm in the cookie specification https://tools.ietf.org/html/rfc6265